css-grid-1/grid-items/grid-item-containing-block-004.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Grid item sizing</title>
```

Comment:
Maybe we could change a little bit the title:
"Grid item sizing in a positioned grid container"